### DISCUSSION TITLE
Requesting to join team

### DISCUSSION THREAD

**Discussion by arslan1400** [ROM-builders/temporary#22548] [General]
My successful builds
https://cirrus-ci.com/build/6131383416913920
https://cirrus-ci.com/build/5753969809555456

Source code to build my ROM
[DT](https://github.com/arslan1400/android_device_samsung_a52sxq)
[VT](https://github.com/arslan1400/android_vendor_samsung_a52sxq)
[kernel](https://github.com/arslan1400/android_kernel_samsung_sm7325)

My github username
[@arslan1400](https://github.com/arslan1400)
Local manifest
[arslan1400/local_manifest](https://github.com/arslan1400/local_manifest)
Telegram username [@Arslan1040](https://t.me/Arslan1040)

**Comment by Apon77** [comment]
please follow #learn instruction from telegram group.
